BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -404,21 +404,9 @@ - (NSString *)fullCollaborationFieldsParameterString
     return [array componentsJoinedByString:@","];
 }
 
-- (NSString *)modelID
+- (NSString *)deviceName
```

Comment:
Maybe deviceModelName?

deviceName is more when you set it to say "Helen's iPhone"